run-qemu/run.sh

```diff
@@ -5,6 +5,26 @@ trap 'exit 2' ERR
 
 source $(cd $(dirname $0) && pwd)/../helpers.sh
 
+foldable start bpftool_checks "Running bpftool checks..."
+bpftool_exitstatus=
+if [[ "${KERNEL}" = 'LATEST' ]]; then
+	# "&& true" does not change the return code (it is not executed if the
+	# Python script fails), but it prevents the trap on ERR set at the top
+	# of this file to trigger on failure.
+	"${REPO_ROOT}/${KERNEL_ROOT}/tools/testing/selftests/bpf/test_bpftool_synctypes.py" && true
```

Comment:
it is not happy with this variable not being set in https://github.com/libbpf/libbpf/runs/8102670715?check_suite_focus=true

Reply:
I think it's fixed now: https://github.com/danielocfb/libbpf/runs/8102983329

Reply:
it failed s390x run for an unrelated reason, I retried and will finish review today